docs/Encyclopedia/pages/Affymetrix_SNP_6.0.md

```diff
@@ -8,7 +8,7 @@ Data from the Affymetrix SNP 6.0 platform is used by the GDC to produce harmoniz
 
 ## References ##
 1. [GDC Copy Number Variation Documentation](/Data/Bioinformatics_Pipelines/CNV_Pipeline/)
-2. [GDC Reference Files](https://gdc.cancer.gov/about-data/data-harmonization-and-generation/gdc-reference-files)
+2. [GDC Reference Files](/about-data/data-harmonization-and-generation/gdc-reference-files)
```

Comment:
The shortening of URLs will only work with things within docs.gdc.cancer.gov.  I think this URL will probably not work. There is no way to prevent this website or the portal links from displaying as "external links".  That will require some change in how the site is built.

Reply:
Fixed file to include base URL